x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/documents.test.tsx

```diff
@@ -32,46 +34,61 @@ describe('Documents', () => {
     expect(wrapper.find(SearchExperience).exists()).toBe(true);
   });
 
-  it('renders a DocumentCreationButton if the user can manage engine documents', () => {
-    setMockValues({
-      ...values,
-      myRole: { canManageEngineDocuments: true },
+  describe('DocumentCreationButton', () => {
+    const getHeader = (wrapper: ShallowWrapper) =>
+      wrapper.find(EuiPageHeader).dive().children().dive();
```

Comment:
I will say the new `EuiPageHeader` is awesome for cleaner source/view code, but it's slightly more annoying to shallow test buttons/actions 😬 not a huuuge deal though

Reply:
Truth!